core/tests/performance_framework/perf_services.py

```diff
@@ -240,8 +240,11 @@ def _setup_driver(self, proxy=None, use_proxy=False):
             chrome_options = webdriver.ChromeOptions()
             # Disable several subsystems which run network requests in the
             # background. This helps reduce noise when measuring network
-            # performance.
+            # performance. Also, disable prerendering by chrome which renders
+            # a page in the background leading to wrong test results.
```

Comment:
One factor to consider is that not all browsers support prerender, so to keep consistency we could go with b. And, also the performance framework wouldn't be able to test the advantage offered by prerender since it tests all pages independently. There are no tests that test sequence of pages visited one after another.